lib/connect-redis.js

```diff
@@ -119,7 +119,7 @@ module.exports = function (session) {
       self.emit('error', er);
     });
 
-    self.client.on('disconnec', function (er) {
+    self.client.on('disconnect', function (er) {
```

Comment:
As it turns out clients don't emit a `disconnect` event, this would need to be `end` or `close`.  Seems like `end` gets emitted when its done retrying (according to the ioredis docs) which seems more appropriate (less disruptive).

Also, neither of these events return an error so we can remove that part.